bot/zephyr.py

```diff
@@ -66,17 +66,15 @@ def source_zephyr_env(zephyr_wd):
     cmd = ['source', './zephyr-env.sh', '&&', 'env']
     cmd = subprocess.list2cmdline(cmd)
 
-    p = subprocess.Popen(cmd, cwd=zephyr_wd, shell=True,
-                         stdout=subprocess.PIPE, executable='/bin/bash')
+    p = subprocess.check_output(cmd, cwd=zephyr_wd, shell=True, executable='/bin/bash')
```

Comment:
`p` would be a process in the 'before' code. Now this is an output.

Reply:
I might overthought this. Using decode() on readlines() output is sufficient